.kitchen.yml

```diff
@@ -21,6 +21,7 @@ platforms:
   - name: opensuse-leap-42
   - name: ubuntu-14.04
   - name: ubuntu-16.04
+  - name: ubuntu-17.04
```

Comment:
Why add 17.04? As far as I know we don't support non-LTS releases of Ubuntu, and the platform matrix here is for chef-client-supported platforms only. I'm not opposed, generally, but it does add more time to the test runs.

For Habitat in particular, I don't think we need to test everything on every Linux platform. The supervisor resource uses systemd, which then requires a platform with systemd installed, but other than that Habitat works on any platform.

Reply:
I added 17.04 primarily because we use that platform over here - https://github.com/chef-cft/habitat_workstation/.

I'm happy to yank it though.